pkg/cluster/volumes.go

```diff
@@ -167,7 +170,7 @@ func (c *Cluster) populateVolumeMetaData() error {
 		return err
 	}
 
-	if len(currentVolumes) != len(c.EBSVolumes) {
+	if len(currentVolumes) != len(c.EBSVolumes) && len(c.EBSVolumes) > 0 {
```

Comment:
On first run c.EBSVolumes will be empty at this point. We already logged the size of known volumes above. Therefore, we should print this debug message only on subsequent sync runs and only if there is a difference.